pkg/kube/volume/volume.go

```diff
@@ -349,10 +349,7 @@ func zoneToRegion(zone string) string {
 
 func zonesToRegions(zone string) []string {
 	reg := map[string]struct{}{}
-	// TODO: gocritic rule below suggests to use regexp.MustCompile but it
-	// panics if regex cannot be compiled. We should add proper test before
-	// enabling this below so that no change to this regex results in a panic
-	r, _ := regexp.Compile("-?[a-z]$") //nolint:gocritic
```

Comment:
This code will panic in any case, because error is ignored, and `r` will be `nil` in case of wrong regex. So, disabling linter and usage of `Compile` did not solve the issue with possible panic.

Anyway, now we do have unit test which will detect panic.